src/main/java/org/deepsymmetry/beatlink/BeatFinder.java

```diff
@@ -252,10 +250,10 @@ public void run() {
     /**
      * Stop listening for beats.
      */
-    public synchronized void stop() {
+    public synchronized void close() {
```

Comment:
There is no need to rename this. Let’s avoid any backwards incompatible changes we possibly can.

Reply:
Done.